docs/changelog.md

```diff
@@ -8,8 +8,23 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 
 ### Added
 
+- [Documentation](guide/angular.md) for using web components with Angular's standalone bootstrapping
+- Added `filter` and `filterPlaceholder` properties to `six-header-dropdown`.
+- Added `caret`, `disabled`, `loading`, `submit` and `reset` properties to `six-header-menu-button`.
```

Comment:
Would be great if you start the changes with the component that has changed and then list them. 

Like we did for example in version 4.2.6  

I think that way the changes are easier to grasp.